themes/default/content/registry/packages/azure-native/from-v2-to-v3/_index.md

```diff
@@ -0,0 +1,250 @@
+
+#### Pending Changes on Default Versions
+
+You may see pending changes when using the default version as the shape of the resource may have changed. You can choose to accept the changes, update your program to modify the resource properties to mitigate changes, or choose an API version which is compatible with the older version of the resource.
+
+A full list of default version changes can be found in the [resource versions table](./resource-versions).
+
+To continue using the previous Azure API version of a resource:
+
+1. Check the resource documentation in your IDE or our [registry API docs](https://www.pulumi.com/registry/packages/azure-native/) which identifies the previous version for each resource. For example: `Azure REST API Version: 2024-10-01. Prior API version in Azure Native 2.x: 2023-05-01`.
+2. Generate and use the local SDK for the previous version of the resource as described in the [version guide](./version-guide).
+
+#### New module structure and naming aligned closer to Azure SDKs
+
+The Azure specification sometimes contains related but distinct services in one namespace. For example, `Microsoft.Network` contains `DNS`, `DnsResolver`, `Frontdoor`, `Network`, `PrivateDns`, and `TrafficManager`.
+
+The Azure SDKs reflect this so that these different network-related services are logically grouped instead of mixed up in one `Network` namespace. In v3 of this provider, we have aligned several modules to match the Azure SDK layout. The changes are:
+
+- `Cache` is split into `Redis` and `RedisEnterprise` which also reflects the current names of the services.
+- `Devices` is split into `DeviceProvisioningServices` and `IotHub`.
+- `DocumentDB` is not split but renamed to `CosmosDB` to reflect the current naming of the service.
+- `Insights` is split into `ApplicationInsights` and `Monitor`.
+- `Network` is split into `Dns`, `DnsResolver`, `Frontdoor`, `Network`, `PrivateDns`, and `TrafficManager`.
+
+This changes require corresponding updates in your Pulumi programs. Existing resources will not be recreated, however, due to the use of aliases.
+
+#### De-duplication of conflicting resources and types
+
+The Azure specification sometimes uses the same name for different resources in the same module. For example, `CosmosDB.FirewallRule` can refer to both regular and MongoDB (`Microsoft.DocumentDB/mongoClusters`) instances. In v3, we have de-duplicated these to `FirewallRule` and `MongoClusterFirewallRule`. The complete list of changes are:
+
+- In `CosmosDB`, `FirewallRule` was split into `FirewallRule` and `MongoClusterFirewallRule`.
+- In `HDInsight`, `Cluster` was split into `Cluster` and `ClusterPoolCluster`.
+- In `HybridContainerService`, `HybridIdentityMetadatum` was split into `HybridIdentityMetadata` and `ClusterInstanceHybridIdentityMetadatum`.
+- In `NetApp`, resources relating to Capacity Pools are prefixed with `CapacityPool`, splitting `Backup` into `Backup` and `CapacityPoolBackup`.
+
```

Comment:
This is probably also worth a similar line to above. If you use these resource, you'll need to update your program files with the new names, but aliases have been added to allow smooth migration for existing resources to the new names.